particles/Renderer.hpp

```diff
@@ -38,21 +38,21 @@ class Renderer : public score::gfx::GenericNodeRenderer
             QRhiShaderResourceBindings* srb);
     QRhiBuffer* particleOffsets{};
     QRhiBuffer* particleSpeeds{};
-    QRhiBuffer* particleSpeedModifier{};
+    QRhiBuffer* particleControls{};
     bool particlesUploaded{};
     QRhiComputePipeline* compute{};
     QRhiTexture* m_texture{};
     bool m_uploaded = false;
-    float data[instances * 4];
-    float speed[instances * 4];
+    float *data = (float *)malloc(maxparticles * 4 * sizeof(float));
```

Comment:
btw is that freed somewhere ? instead consider using 

     std::make_unique<float[]>(maxparticles * 4);

and storing the data in a `std::unique_ptr<float[]>` (or std::vector<float> could work too)